p11-kit/util.c

```diff
@@ -236,3 +237,22 @@ _p11_get_progname_unlocked (void)
 		return NULL;
 	return p11_my_progname;
 }
+
+/**
+ * p11_kit_check_version:
+ * @major: major version
+ * @minor: minor version
+ * @micro: micro version
+ *
+ * Check if the run-time version of p11-kit meets the requirement
+ * given by a triple: @major, @minor, and @micro.
+ *
+ * Returns: %CKR_OK if the version requirement meets; %CKR_ARGUMENTS_BAD otherwise.
+ * Since: 0.25.4
+ */
+CK_RV
```

Comment:
Would it be better if the return value was `bool` as it literally returns a binary state?

Reply:
Afaik we intentionally do not expose `bool` from the public API for compatibility reasons.

Reply:
how about int?

Reply:
Yeah, we use `int` in many places so we can do that here too.